commands.json

```diff
@@ -22,7 +22,8 @@
     "less",
     "grep",
     "rm",
-    "unalias"
+    "unalias",
+    "which"
     ],
   "globalExclusions": [
     "alias",
```

Comment:
Which should be able to be global too, which you can simply do by adding it lower on as well, down here:

https://github.com/dthree/cash/blob/master/commands.json#L171